header-generator/translate.go

```diff
@@ -193,7 +193,7 @@ var (
 	Uint16Translator                         = &ScalarConverter{"uint16_t", "C.uint16_t", "uint16"}
 	Uint32Translator                         = &ScalarConverter{"uint32_t", "C.uint32_t", "uint32"}
 	Uint64Translator                         = &ScalarConverter{"uint64_t", "C.uint64_t", "uint64"}
-	SizeTranslator                           = &ScalarConverter{"size_t", "C.uint64_t", "uint64"}
+	SizeTranslator                           = &ScalarConverter{"size_t", "C.size_t", "uint64"}
```

Comment:
I'm still a little worried about this, since it seems similar to what caused me [this mental mistake](https://github.com/golang/go/issues/50100) in the past. It is possible that vks will read past the end of size_t when size_t is defined as less than 64 bytes. But since that problem existed before this change, I'll merge and open an issue on myself about it.

Reply:
Actually, nevermind. I was thinking of the type converter. Go should handle the scalar converter fine. The scalar converter doesn't do any of that unsafe pointer stuff anymore.